en/advanced-topics/java-reflection.md

```diff
@@ -116,21 +115,23 @@ public class AppActivity extends Cocos2dxActivity {
 }
 ```
 
-Then we call `showAlertDialog` in JavaScript:
+Next, calling `showAlertDialog` in JavaScript:
 
 ```js
 jsb.reflection.callStaticMethod("org/cocos2dx/javascript/AppActivity", "showAlertDialog", "(Ljava/lang/String;Ljava/lang/String;)V", "title", "hahahahha");
 ```
 
-You should see a Android native AlertDialog now.
+An Android native AlertDialog should show now.
 
 ## One more thing
 
-Now we have successfully called Java methods in JavaScript, so can we call JavaScript in Java? Of course!
+Now that it is possible to successfully called Java methods in JavaScript, is it possible to call JavaScript in Java? Of course!
+
+The engine contains class `CocosJavascriptJavaBridge`, which has a `evalString` method that can execute JavaScript, and is located in the `resources\3d\cocos2d-x-lite\cocos\platform\android\java\src\com\cocos\lib\CocosJavascriptJavaBridge.java` file in the engine directory. Please note that this time JavaScript code should be run in the gl` thread.
```

Comment:
gl\` -> \`gl\`

Reply:
Here I don't think the original sentence should be split up.

Reply:
I moved it out so that the example code block has a proper description above it.